modules/uhttp/README.md

```diff
@@ -27,12 +28,18 @@ func main() {
 }
 
 func registerHTTP(service service.Host) []uhttp.RouteHandler {
-  handleHome := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+  handleHome := http.HandlerFunc(func(ctx fx.Context, w http.ResponseWriter, r *http.Request) {
+    ctx.Logger().Info("Inside the handler")
     io.WriteString(w, "Hello, world")
   })
 
   return []uhttp.RouteHandler{
     uhttp.NewRouteHandler("/", handleHome)
   }
 }
+
+HTTP handlers are set up with filters that inject tracing, authentication information etc. into the
```

Comment:
This belongs outside the cod block (below line 45). See the readme as merged: https://github.com/uber-go/fx/tree/master/modules/uhttp